src/stdio_mgr/stdio_mgr.py

```diff
@@ -34,14 +34,19 @@
 
 
 class _PersistedBytesIO(BytesIO):
-    """Class to persist the stream after close.
+    """Class to persist the value after close.
 
-    A copy of the bytes value is available at :attr:`_closed_buf` after
+    A copy of the bytes value is given to a callback prior to
```

Comment:
It's going to be a copy of the bytes value as a `str`, though, right?  IIRC `.getvalue()` does a `.decode()`.

Reply:
`_PersistedBytesIO.getvalue` is bytes.  https://docs.python.org/3/library/io.html#io.BytesIO.getvalue